app/static/css/style.css

```diff
@@ -208,5 +208,37 @@ a.link-text:hover {
     width: 100%;
     max-width: 100%;
     max-height: 100%;
-    margin-top: 2rem;
+    margin-top: 1rem;
 }
+
+.btn-primary {
+    background-color : var(--p-blue);
+    /* margin-top: 1rem; */
```

Comment:
Do we need this commented out code? If not, please remove it.

Reply:
Removed